langchain/callbacks/manager.py

```diff
@@ -95,7 +104,34 @@ def _handle_event(
                 getattr(handler, event_name)(*args, **kwargs)
         except Exception as e:
             # TODO: switch this to use logging
-            print(f"Error in {event_name} callback: {e}")
+            logging.warning(f"Error in {event_name} callback: {e}")
+
+
+def _handle_chat_start_event(
+    handlers: List[BaseCallbackHandler],
+    serialized: Dict[str, Any],
+    messages: List[List[BaseMessage]],
+    *args: Any,
+    **kwargs: Any,
+) -> None:
+    """Handle the chat start event."""
+    for handler in handlers:
```

Comment:
why add another set of methods? can't we just check `event_name` from within `_handle_event` and `_ahandle_event`? Feel like that would be cleaner

Reply:
I don't feel strongly. Felt that adding this was neater than adding a catch for not implemented collecting `*args` then falling back if the event_name looks right. But I can combine